gnuboardProject/frontend/src/components/Portfolio/PDFExport/PDFExportModal.tsx

```
@@ -0,0 +1,49 @@
+              섹션
+            </label>
+            <label>
+              <input type="checkbox" name="skills" />
+              기술 스택
+            </label>
+          </div>
+        </div>
+
+        <div className="modal-actions">
+          <button onClick={onClose}>취소</button>
+          <button onClick={() => onExport([])}>변환</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)